gateway/src/apicast/oauth/oidc.lua

```diff
@@ -194,8 +196,17 @@ function _M:transform_credentials(credentials, cache_key)
   end
 
   local payload = jwt_obj.payload
-
   local app_id = payload.azp
+
+  if self.service and self.service.oidc and self.service.oidc.client_id then
+    local tmpl, tmpl_err = TemplateString.new(self.service.oidc.client_id, "liquid")
```

Comment:
I see that this is always interpreted as liquid. I'm not sure that's a good idea from a UX perspective. That forces the user to write something like `{{ aud }}` in the UI instead of just `aud`.